decentraland/unity-explorer
# fix: Billboard mode selection

## What does this PR change?

Fixes #1334 

Updates the billboard mode so it looks towards the camera, as it did in [unity-renderer](https://github.com/decentraland/unity-renderer/blob/f782a1245d8737db2eeaad1939f3fbc17749f4f1/unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Defaults/PBBillboard_Defaults.cs#L5)

## How to test the changes?


Recomendation: In Debug Menu, in Landscape setting, move the load raidus to 1.

Clear local asset bundle cache
Go to Goerli
Go to the zombie scene (78,-1). The health billboard above zombies should look at you, and go down when you click it. The zombie should dissapear when reaching 0

IMPORTANTE: The zombie animation is broken, and not fixed in this PR. You can disregard it

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
